cup.css

```
@@ -0,0 +1,56 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background: #262626;
+}
+body{
+    height: 100vh;
+    width: 100%;
+    display: grid;
+    place-items: center;
+}
+
+.cup{
+    width: 200px;
+    height:220px;
+    
+    border: 6px solid #fff;
+    border-top: 2px solid transparent;
+    border-radius: 15px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    
+    box-shadow: 0 0 0 6px #fff , 0 20px 35px rgba(0,0,0,1);
+
+    background: url(https://image.ibb.co/fmHm66/wave.png);
+    background-repeat: repeat-x;
+    
+    animation: animate 10s linear infinite;
+
+}
+.handle{
+    position: relative;
+    right: -45px;
+    top:50px;
+    
+    transform: translate(150px,-20px);
+    
+    border: 6px solid #fff;
+    border-top-right-radius:50% ;
+    border-bottom-right-radius:50% ;
+}
+
+
+@keyframes animate{
+    0%{
+        background-position: 0 0;
+    }
+    25%{
+        background-position: 0 80px;
+    } 
+
+    100%{
+        background-position: 500px -40px;
+    }
+}
```

```suggestion
@keyframes animate{
   0% {
    background-position: 0 0;
  }

  10% {
    background-position: 212px 0px;
  }
  20% {
    background-position: 424px 0px;
  }
  30% {
    background-position: 636px 80px;
  }
  40% {
    background-position: 848px 160px;
  }

  50% {
    background-position: 1060px 160px;
  }
  60% {
    background-position: 1272px 80px;
  }
  70% {
    background-position: 1484px 40px;
  }
  80% {
    background-position: 1696px 0px;
  }
  90% {
    background-position: 1908px 0px;
  }

  100% {
    background-position: 2120px 0px;
  }
}
```

suggestion number (2 of 2) to create smooth animation